### DISCUSSION TITLE
Creating users without an email address

### DISCUSSION THREAD

**Discussion by SpctrX** [pelican-dev/panel#149] [Ideas]
Hi,
I've thought about creating users without an email address.
I'm using this type of panel for my project, where some users need access to some services and I would recommend to create there user accounts where I don't have to ask for the email addresses first that I can just create the user with a name and password. Maybe the requirements can be enabled/disabled in the administration?

Maybe someone finds this idea usefull as well

**Comment by lancepioch** [comment]
Thanks, this will be supported in the near future and is planned!
